content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,225 @@
+local Istio control plane and Istio remote on `cluster-1` and `cluster-2`.
+
+This example uses `cluster-1` as local Istio control plane and `cluster-2` as Istio remote.
+
+## Deploy Bookinfo Example Across Clusters
+
+__NOTE__: The following example enables [sidecar automatic injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
+
+1.  Install bookinfo on the `cluster-1`. Remove the `reviews-v3` deployment to deploy on remote:
+
+    {{< text bash >}}
+    $ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+    $ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+    $ kubectl delete deployment reviews-v3
+    {{< /text >}}
+
+1.  Create the `reviews-v3.yaml` manifest for deployment on the remote:
+
+    {{< text yaml plain "reviews-v3.yaml" >}}
+    ---
+    ##################################################################################################
+    # Ratings service
+    ##################################################################################################
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: ratings
+      labels:
+        app: ratings
+    spec:
+      ports:
+      - port: 9080
+        name: http
+    ---
+    ##################################################################################################
+    # Reviews service
+    ##################################################################################################
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: reviews
+      labels:
+        app: reviews
+    spec:
+      ports:
+      - port: 9080
+        name: http
+      selector:
+        app: reviews
+    ---
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: reviews-v3
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: reviews
+            version: v3
+        spec:
+          containers:
+          - name: reviews
+            image: istio/examples-bookinfo-reviews-v3:1.5.0
+            imagePullPolicy: IfNotPresent
+            ports:
+            - containerPort: 9080
+    EOF
+    {{< /text >}}
+
+    _Note:_ The `ratings` service definition is added to the remote cluster because `reviews-v3` is a
+    client of `ratings` and creating the service object creates a DNS entry.  The Istio sidecar in the
+    `reviews-v3` pod will determine the proper `ratings` endpoint after the DNS lookup is resolved to a
+    service address.  This would not be necessary if a multicluster DNS solution were additionally set up, e.g. as
+    in a federated Kubernetes environment.
+
+1.  Install the `reviews-v3` deployment on the remote `cluster-2`.
+
+    {{< text bash >}}
+    $ kube apply -f $HOME/reviews-v3.yaml
+    {{< /text >}}
+
+1.  Get the `istio-ingressgateway`'s `INGRESS_HOST` and `INGRESS_PORT` variables for accessing the gateway via
```

Comment:
`[Determine  the ingress IP and ports](link) for `istio-ingressgateway`s ...   end goal is to get rid of "these instructions" and replace it  with something more descriptive.

Reply:
Done